.github/workflows/build-dotnet-dynamic.yml

```
@@ -0,0 +1,169 @@
+
+  tests:
+    name: Run Tests
+    uses: ./.github/workflows/step-dotnet-tests.yml
+    needs:
+      - commitlinter
+      - version
+      - collect-matrix
+    strategy:
+      fail-fast: ${{ inputs.failFast || true }}
+      matrix:
+        test-filter: ${{ fromJson(needs.collect-matrix.outputs.matrix) }}
```

actionlint (1.7.9)

property "matrix" is not defined in object type {}

(expression)